src/App.js

```diff
@@ -1,7 +1,23 @@
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import LandingPage from "./components/LandingPage/LandingPage";
+import ContactPage from "./components/ContactPage/ContactPage";
+import MenuPage from "./components/MenuPage/MenuPage";
+import AdminPanel from "./components/AdminPage/AdminPanel/AdminPanel";
+import AdminHome from "./components/AdminPage/AdminHome/AdminHome";
+
 function App() {
   return (
-    <div className="App">
-    </div>
+    <Router>
+      <div className="App">
+        <Routes>
```

Comment:
To create routing, you can also create an array of objects containing fields such as route and element and then map on it. But your approach is ok.